### ISSUE TITLE
podfileLinker in rnn-link does not update Podfile min platform

### ISSUE THREAD

**Issue by Andarius** [wix/react-native-navigation#6636]
### Issue Description

When running **rnn-link**, `platform :ios, '10.0'` is not updated inside the **Podfile** to match min specs of React Native Navigation causing an error during the pod install.

### Environment
* React Native Navigation version: ^7.1
* React Native version: 0.63.2
* Platform(s) (iOS, Android, or both?): iOS
* Device info (Simulator/Device? OS version? Debug/Release?): -

**Comment by Andarius** [comment]
I guess that something like this should do

```
var fs = require("fs");

const MIN_VERSION = 11

const updateVersion = function(file){
	const versionMatch = file.match(/platform :ios, '(\d+\.\d+)'/);
	if(versionMatch && parseFloat(versionMatch[1]) < MIN_VERSION){
		const newVersion = versionMatch[0].replace(versionMatch[1], MIN_VERSION.toString());
		file = file.replace(versionMatch[0], newVersion);
	}
	return file;
}


let file = fs.readFileSync('./ios/Podfile', 'utf8');
const fileUpdated = updateVersion(file);
console.log(fileUpdated)
```

**Comment by danilobuerger** [comment]
I don't really like the idea of messing with the Podfile. What if I already have multiple targets with multiple platforms in there? The only time that his would make sense is on a fresh install. But if I add rnn later, this is just asking for trouble.

**Comment by Andarius** [comment]
I agree, otherwise update the documentation to specify the minimum version in Podfile ? https://wix.github.io/react-native-navigation/docs/installing/#installing-with-npx-rnn-link

**Comment by danilobuerger** [comment]
Sure, we are happy to accept a PR for that!

**Comment by Andarius** [comment]
@danilobuerger  https://github.com/wix/react-native-navigation/pull/6637

**Comment by guyca** [comment]
@Andarius @danilobuerger @yogevbd How can we fix/workaround this issue? Updating the instructions in the manual installation guide is a good step, but we should really avoid any manual steps when integrating RNN with a project.

What are our options here?
1. Restore support for iOS 10. This can be only for compilation reasons, it's totally fine if features will be broken or won't work on iOS 10.
2. Update the pod file in the link script. The link script doesn't commit changes and it's up to the user to confirm and commit the changes made by the script. We're already modifying AppDelegate, whether the app is a fresh project or an existing one. Seems like we should be fine modifying the podfile, no?

**Comment by danilobuerger** [comment]
Well if its one or the other, we should use the 2. option. The first one will just add way more friction and wasted developer hours down the road.

**Comment by guyca** [comment]
I agree. @Andarius lets move forward ASAP with your initial suggestion outlined in https://github.com/wix/react-native-navigation/issues/6636#issuecomment-702722886 💪

**Comment by danilobuerger** [comment]
Fixed in #6674
